services/home/docs/demos/publisher-ad-quality-req.md

```diff
@@ -0,0 +1,224 @@
+the product tag to exclude.
+
+### Assumptions
+
+- The SSP has a database relating ad URLs to ad metadata & this information is available in the K/V service.
+- We assume the SSP has deployed a Key/Value service (BYOS or TEE).
+
+### Key Exclusions
+
+- Dynamic updates of K/V pairs coming from the SSP backend (we will use static data for this demo).
+- Excluding ads based on DSP supplied meta-data.
+
+### System Design
+
+Using Protected Audience API, the user visits a shopping site, and gets added to an interest group. Later the same user visits a news site. Their
+browser runs an on-device auction & bidding logic will select the winning ad which will be shown to the user. The user now presses one of several
+buttons designed to exclude ads with certain product tags. The news page is refreshed, adding a url parameter for the excluded product tags. The new
+ad auction will receive key / value meta-data from the SSP, indicating the product tags of the ad. If it matches the excluded product tag set by the
+user, the ad will be excluded from the auction, and the user will see no ad.
+
+#### User Journey
+
+```mermaid
+sequenceDiagram
+Title: Enforcing publisher ad requirements in PAAPI using K/V
+
+
+participant Browser
+participant Publisher
+participant SSP
+participant Advertiser
+participant DSP
+
+
+Browser->>Advertiser:Visits shop site and views product
+Advertiser->>Browser:Adds DSP tag on the page
+Browser->>DSP:Browser requests scripts from DSP
+DSP->>Browser:Call joinAdInterestGroup() with the config
+
+
+
+
+Browser->>Publisher:Visits news site
+note right of Browser: With optional "?excludeCreativeTag=" param, value e.g. "redShoe"
+Publisher->>Browser: Adds SSP tag on the page
+SSP->>Browser: Browser loads scripts from SSP, incl. auction config with trustedScoringSignalsURL
+Browser->>Browser: Adds sellerSignals to auction config, incl. excluded product tag.
+SSP->>Browser:SSP calls runAdAuction with the config
+
+
+
+
+Browser->>SSP:Get Key/Values
+note right of Browser: Keys are ad urls
+SSP->>Browser:Returns Keys/Values
+note right of Browser: Response like: {“product_tags”: [“redShoe”, “shortsShoe”]}
+
+
+Browser->>Browser:scoreAd(...)
+note right of Browser: Uses the product tags for the ad from the K/V signal & the excludeCreativeTag from the seller signals.
+note right of Browser: Scores 0 if ad product tags include excludeCreativeTag.
+
+
+Browser ->> Browser: Return ad in the runAdAuction call if there’s a winner
+SSP ->> Browser: Set iframe src attribute or fenced frame config property with the ad
+Browser ->> DSP: Fetch ad
+Browser ->> Browser: Render ad
+```
+
+[Full-sized diagram](./img/publisher-ad-quality-req-flow.png)
+
+</TabItem>
+<TabItem value="demo" label="Demo">
+
+## Demo
+
+### Prerequisites
+
+- Chrome > v128 (Open chrome://version to look up your current version)
+
```

Comment:
Suggest adding : 
- Enable Privacy Sandbox Protected Audience API (Open chrome://settings/adPrivacy and enable *Site-suggested ads*)

Reply:
Done